sklearnex/cluster/tests/test_kmeans.py

```diff
@@ -27,16 +28,23 @@
 
 @pytest.mark.parametrize("dataframe,queue", get_dataframes_and_queues())
 def test_sklearnex_import(dataframe, queue):

```

Comment:
This is not strong enough test coverage

Reply:
@md-shafiul-alam please add more test suits with `get_dataframes_and_queues`